lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -814,14 +820,17 @@ def _GetFd(self, fd, fdname):
       logging.info("Passing fd %s via SCM_RIGHTS failed: %s", fd, err)
       raise
 
-  def _AddFd(self, fd):
+  def _AddFd(self, fd, fdset_id=None):
```

Comment:
Too bad a fdset can't be created empty, this way the call sequence looks a bit ugly…